src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Ctor.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Ctor
+    {
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _arrayBufferWriter = new ArrayBufferWriter<byte>();
+        }
+
+        [Benchmark]
+        public void Ctor()
+        {
+            var state = new JsonWriterState(options: new JsonWriterOptions { Indented = Formatted, SkipValidation = SkipValidation });
+            var json = new Utf8JsonWriter(_arrayBufferWriter, state);
```

Comment:
to prevent from dead code elimination you need to return the created object from the benchmark. Then BDN consumes it using [Consumer](https://github.com/dotnet/BenchmarkDotNet/blob/master/src/BenchmarkDotNet/Engines/Consumer.cs)

Reply:
Currently, it doesn't look like it is being eliminated, but sure, we can return it.

Reply:
> Currently, it doesn't look like it is being eliminated, but sure, we can return it

Sorry, I was not clear. It's a best practice that makes sure it does not happen. In this case, it most probably does not. BDN would report 0ns as the time and complain about it with warning